data/src/main/java/org/oppia/android/data/backends/gae/model/GaeFeedbackReport.kt

```diff
@@ -4,7 +4,7 @@ import com.squareup.moshi.Json
 import com.squareup.moshi.JsonClass
 
 /** Data class for the feedback report sent by the Android app to remote storage. */
-// TODO(#2801): Link backend domain model
+// TODO(#3636): Link backend domain model
```

Comment:
@jcqli won't this be addressed by your pending PR?

Reply:
@BenHenning Just adding this thread as we have received confirmation from @jcqli.
I have created a new issue according to this conversation i.e #3636.
https://github.com/oppia/oppia-android/pull/3508#issuecomment-890165282

Reply:
Sorry to be clear: I'm fairly certain this is being addressed by #2878 & https://github.com/oppia/oppia/pull/12367. Rather than introducing a new issue, why not just use #3016 since that's already being tracked?

Reply:
@jcqli Can you confirm if I should update this TODO to use issue no. #3016 and keep the description the same as before.

Reply:
Ah I see what you mean -- I originally didn't want #2878 to be blocked by oppia/oppia#12367 but I think that will be fine. Let's use #3016 and it will be fixed in #2878.

Reply:
Done. Thanks.

Reply:
Thanks @jcqli!